#######################
## Robot Arm Config ###
#######################

robot_arm: "panda"
eff_home_pos: [0.5, 0, 0.6] # x, y, z for end-effector
default_joints: [0, -0.7853981633974483, 0, -2.356194490192345, 0, 1.5707963267948966, 0.7853981633974483] #[0, -pi/4, 0, -3*pi/4, 0, pi/2, pi/4]
arm_group: "bob_arm"
base_link: "bob_link0"
effector_link: "bob_link8"
eff_default_orien: [0.9237820580963242, -0.38291837679374485, 7.93796317786616e-05, 0.0004685635894043005]


##########################
## Robot Gripper Config ##
##########################
robot_gripper: "panda"

# CAUTION!!! following value needs to be calibrated
g2e_offset: 0.133 



###################
## Camera Config ##
###################
camera: "realsense"
color_resol: [848, 480]
depth_resol: [848, 480]
depth2color_align:
  offset: [-10, -14] #H, W
  scale: [0.72, 0.72] # H, W


# CAUTION!!! following values need to be calibrations using hand-eye-calibration
camera_setup: "eye-on-hand"
eff2camera_pos:
  pose: [-0.12, -0.125, -0.035]
  orien: [7.93796318e-05, -4.68563589e-04, -9.23782058e-01,  3.82918377e-01] ## target will be [0, 1, 0, 0]

load_intrinsic: "from_camera"



####################
## MoveIt Setting ##
####################

ros_sleep_time: 0.2
max_vel_scale: 0.3
max_acc_scale: 0.1
plan_pipe_id: 'pilz_industrial_motion_planner'
planner_id: 'PTP'

############################
## Pick and Place Setting ##
############################

pick_raise_offset: 0.05
place_raise_offset: 0.03
crop_scale: 0.9